src/Benchmarks/Program.cs

```diff
@@ -97,10 +99,28 @@ public static void Main(string[] args)
                         }
                     });
                 }
-                else if (string.Equals(kestrelTransport, "Sockets", StringComparison.OrdinalIgnoreCase))
+                else if (string.Equals(kestrelTransport, "Sockets", StringComparison.OrdinalIgnoreCase)
+                         && threadPoolDispatching != true)
                 {
                     webHostBuilder.UseSockets();
                 }
+                else if (string.Equals(kestrelTransport, "RHLinux", StringComparison.OrdinalIgnoreCase))
+                {
+                    if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                    {
+                        throw new PlatformNotSupportedException("RHLinux Transport requires Linux");
+                    }
+#if NETCOREAPP2_0
```

Comment:
The app targets `<TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>`.
And the library targets `netstandard2.0`.
The netstandard2.0 library doesn't work on the net46 target (net461 needed for netstandard2.0).
I think I can make this work on netstandard1.3. Then this condition (+ matching one in csproj) can go.